content/webapp/components/EventSchedule/EventBookingButton.tsx

```diff
@@ -1,36 +1,37 @@
 import { Event } from '../../types/events';
-import { FunctionComponent, Fragment } from 'react';
+import { FunctionComponent } from 'react';
 import ButtonSolidLink from '@weco/common/views/components/ButtonSolidLink/ButtonSolidLink';
 import Message from '@weco/common/views/components/Message/Message';
 import { font } from '@weco/common/utils/classnames';
 import Space from '@weco/common/views/components/styled/Space';
 import styled from 'styled-components';
 import { ticketAvailable, email } from '@weco/common/icons';
+import { isUndefined } from '@weco/common/utils/type-guards';
 
 type Props = {
   event: Event;
 };
 
-function getButtonMarkup(event: Event) {
-  if (!event.eventbriteId) return;
+const BookingButtonLink: FunctionComponent<Props> = ({ event }) => {
+  if (isUndefined(event.eventbriteId)) return null;
 
   if (event.isCompletelySoldOut) {
     return <Message text="Fully booked" />;
   } else {
     return (
       <div data-eventbrite-ticket-id={event.eventbriteId}>
         <ButtonSolidLink
-          link={`https://www.eventbrite.com/e/${event.eventbriteId || ''}/`}
```

Comment:
We know that `event.eventbriteId` is `string`, not `undefined` here – otherwise the function has already returned. We can skip providing a fallback value (here and elsewhere in this PR).